content/examples/shortcodes/search/en.md

```diff
@@ -8,10 +8,10 @@ With this shortcode you can embed a search results widget on the page.
 ## Example
 
 ```
-{{</* search contentType="How-to Guides" showContentTypes=false tags="encodings,commented cue" */>}}
+{{</* search contentType="How-to Guides" showContentTypes=false tags="encodings,commented cue" searchTerm="json" */>}}
```

Comment:
Please change this example to have a multi-word `searchTerm` value instead of `json`. The search string `"built-in function"` would be compatible with the tags and content type. This is so that an author sees an example of how to encode a string that needs to be quoted **inside** a quoted Hugo parameter.

NB This isn't asking for the existing `json` string to be updated to `built-in function`. It's asking for an example of triggering Algolia's [advanced syntax](https://www.algolia.com/doc/api-reference/api-parameters/advancedSyntax/) to be present on the page, where the multi-word search string needs to be an exact match for a phrase on the page.